### ISSUE TITLE
Linux: not possible to navigate between menus using keyboard

### ISSUE THREAD

**Issue by ristomatti** [Microsoft/vscode#1569]
Navigating between File/Edit/View/Goto/Help using left and right arrow keys does not work. I can reproduce the problem in at least Windows 8 and Linux Mint Cinnamon 17.2.
- navigation using arrow keys works in submenus
- on Windows it is possible to switch between the menus using mnemonics (ALT-e to switch from File to Edit) but on Linux even this isn't working

![image](https://cloud.githubusercontent.com/assets/9029939/11951545/5e610a66-a89a-11e5-8cba-77fd7a9ed578.png)

**Comment by jimfoye** [comment]
Related: if you press and release the Alt key, the menu does not retain focus, as it does in standard Windows applications. Normally you can press the Alt key, then press other keys to begin navigating the menu. Pressing the Alt key again would toggle focus away from the menu and back to whatever originally had it. In Code, as soon as you release the Alt key, the menu lose focus.

**Comment by kevin-brown** [comment]
Is there an upstream ticket attached to this, or was this just given the label "upstream" because VS Code is using Electron's application menus?

**Comment by bpasero** [comment]
https://github.com/electron/electron/issues/2504

**Comment by hpg-mheck** [comment]
Valid.  This would flunk 508(c) testing.

**Comment by hpg-mheck** [comment]
Also, having a two-year-old accessibility bug that I can assure you will be obvious within 30 seconds of a vision-impaired user attempting to use the product is excellent evidence of a lack of interest in accessibility in the product.  If there is, in fact, a lack of interest in accessibility in the product, please be well aware that demonstrating that across _several_ products is strategically dangerous for a company selling software to the federal government.  Keyboard navigation is still the quick litmus test when evaluating whether or not a product can or cannot be used for many agencies.

**Comment by sbatten** [comment]
note that this issue has been resolved in the new custom menu bar. it can be enabled via the setting `"window.titlebarStyle": "custom"` and will be default on Windows soon. It can still be enabled on Linux, though is not the default currently mainly due to styling.

**Comment by bpasero** [comment]
Closing, given the workaround.
